core/util.rs

```diff
@@ -1,7 +1,6 @@
-pub fn normalize_ident(ident: &str) -> String {
+pub fn normalize_ident(mut ident: &str) -> String {
```

Comment:
Will merge, but small stylistic nit: probably better to do `let mut ident = ident;` instead of making the parameter `mut`. Can confuse the reader into thinking you're mutating the `ident`.

Reply:
Do you like this better? https://github.com/penberg/limbo/pull/89